Task2.1/Program.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._1
+{
+   
+        public class StringAsCharArray
+        {
+            public char[] _myString;
+
+            public StringAsCharArray(char[] myString)
+            {
+                _myString = myString;
+            
+            }
+            public bool IsDifferentOrEquals(char[] firstString, char[] otherString)
```

Comment:
IsDifferentOrEquals в моем понимании означает, что метод должен возвращать true, если строки разные или одинаковые. Я бы посоветовал назвать метод просто Equals. А так как этот метод экзеплярный, то наверно правильней будет оставить один аргумент otherString и стравнивать его с текущим экземпляром
```suggestion
            public bool Equals(char[] otherString)
```